draft-ietf-rats-corim.md

```diff
@@ -825,6 +825,21 @@ $version-scheme /= int / text
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/8
 
+The following describes details the security version number (svn) and the minimum security version number (min-svn).
+A security version number is used to track changes to an object that are security relevant.
```

Comment:
we should precisely define what we mean by "object" here (or possibly use another, more intuitive, term)

Reply:
I'm open to suggestions. We use Target Environment and `environment-map` to refer to the component that is being attested. The Target Environment could have multiple measurements such as `digest` and `raw-value-group` but these measurements could be describing different objects within the Target Environment. I don't think we have terminology for the object from which a measurement is collected. I like "object" because it is neutral. Alternatives like "component", "device", "module" all have baggage. The O in OID can mean a variety of things without too much baggage. Object does however have baggage with regard to triples structure, but it seems this is obvious when in triples-map context.

Reply:
Maybe we could provide one or more examples of objects, e.g.:
```
A security version number is used to track changes to
an object (e.g., OBJECT_INSTANCE_1, OBJECT_INSTANCE_2)
that are security relevant.
```
which avoids both defining it or finding another term.

Reply:
Fixed in branch - this thread is outdated.